addons/auth_signup/models/res_users.py

```diff
@@ -195,8 +195,9 @@ def action_reset_password(self):
         for user in self:
             if not user.email:
                 raise UserError(_("Cannot send email: user %s has no email address.") % user.name)
+            # TDE FIXME: make this template technical (qweb)
             with self.env.cr.savepoint():
-                template.with_context(lang=user.lang).send_mail(user.id, force_send=True, raise_exception=True)
```

Comment:
but why? :cry:

Reply:
Because lang can be defined on template itself.

Reply:
So shouldn't it be added in this PR as well ?

Reply:
It is already done normally.

Reply:
I added some forgotten lang in various templates recently https://github.com/odoo/odoo/commit/678e04c19dbfefaadd8ab6207b0bb560a3d3bfe4

Reply:
oh, it was hidden in the bottom of the email template, I will now go kill myself by cheese overdose
https://github.com/odoo/odoo/blob/f8ff7debdc86ee910af881be82f7b67201bcd791/addons/auth_signup/data/auth_signup_data.xml#L97

Reply:
Even when you are obviously mistaken you are 100% yourself :lollipop: